src/modules/service-catalog/components/service-catalog-list-item/Search.tsx

```diff
@@ -13,6 +14,10 @@ const StyledField = styled(Field)`
   }
 `;
 
+const StyledMediaInput = styled(MediaInput)`
+  padding-right: ${(props) => props.theme.space.base * 7}px;
```

Comment:
I just realized that this doesn't work for RTL languages. We should use `padding-inline-end` instead